ansible/docker/rally/Dockerfile

```diff
@@ -21,8 +21,10 @@ RUN cd rally_ovn_scale_test \
     && ./install_rally.sh
 
 # Install OVN scale test plugin for rally
-RUN git clone https://github.com/openvswitch/ovn-scale-test.git
+# TODO (huikang): switch to upstream repo after this PR merged
+RUN git clone https://github.com/huikang/ovn-scale-test.git
```

Comment:
Since I changed some code in rally-ovs and the rally-ovs will be installed in the container image, I am not sure if travis-ci will do a merge first and then run the CI job. Otherwise, the ```docker built``` command will pull the rally-ovs from the master, which does not include the fixes to rally-ovs,
In addition, I leave a TODO note so that we can change this after the PR getting merged.